ansible/files/sonic_example_ixia_link.csv

```diff
@@ -1,4 +0,0 @@
-StartDevice,StartPort,EndDevice,EndPort,BandWidth,VlanID,VlanMode
-example-s6100-dut-1,Ethernet0,example-ixia-1,Card1/Port1,40000,,Access
-example-s6100-dut-1,Ethernet1,example-ixia-1,Card1/Port2,40000,,Access
-example-s6100-dut-1,Ethernet2,example-ixia-1,Card1/Port3,40000,,Access
```

Comment:
@selldinesh , Is this format same for UHD100G32 as well ?

Reply:
you can specify as localuhd/18 or Card3/Port2 ... This is example for port 18 in UHD.